src/main/java/core/basesyntax/service/impl/StorageBalanceServiceImpl.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.StorageUpdateService;
+
+public class StorageBalanceServiceImpl implements StorageUpdateService {
```

Comment:
The prefix "Storage" repeats in other names of the implementations. I think it makes the code less readable.